src/components/menus/IconMenu.tsx

```diff
@@ -96,7 +105,35 @@ const IconMenu = ({
                         aria-haspopup="true"
                         aria-expanded={open ? 'true' : undefined}
                         onClick={handlers.click}
-                        sx={{ color: (theme) => theme.palette.text.primary }}
+                        disabled={disabled}
+                        sx={
+                            outlinedButton
+                                ? {
+                                      'borderRadius': 2,
+                                      'border': (theme) =>
+                                          primaryColoredOutline[
+                                              theme.palette.mode
+                                          ],
+                                      'color': (theme) =>
+                                          theme.palette.primary.main,
+                                      '&:hover': {
+                                          border: (theme) =>
+                                              primaryColoredOutline_hovered[
+                                                  theme.palette.mode
+                                              ],
+                                      },
+                                      '&.Mui-disabled': {
+                                          border: (theme) =>
+                                              primaryColoredOutline_disabled[
+                                                  theme.palette.mode
+                                              ],
+                                      },
+                                  }
```

Comment:
Initially I was okay but I think I have changed my mind.

I know this is currently not something that is shared and does not require light/dark mode... but it kinda _feels_ like something that should be in the theme file to be shared.

Mainly gauging this on just how much customization there is here and how it references the colors as well.